user/api.go

```diff
@@ -164,14 +165,20 @@ func (h varsHandler) ServeHTTP(res http.ResponseWriter, req *http.Request) {
 }
 
 func (a *Api) GetStatus(res http.ResponseWriter, req *http.Request) {
+	var s status.ApiStatus
 	if err := a.Store.Ping(); err != nil {
 		a.logger.Println(http.StatusInternalServerError, STATUS_GETSTATUS_ERR, err.Error())
-		res.WriteHeader(http.StatusInternalServerError)
-		res.Write([]byte(err.Error()))
-		return
+		s = status.NewApiStatus(http.StatusInternalServerError, err.Error())
+	} else {
+		s = status.NewApiStatus(http.StatusOK, "OK")
+	}
+	if jsonDetails, err := json.Marshal(s); err != nil {
+		log.Printf("Error marshaling StatusApi data [%s]", s)
+		http.Error(res, "Error marshaling data for response", http.StatusInternalServerError)
+	} else {
+		res.WriteHeader(s.Status.Code)
```

Comment:
Shouldn't we set the content type to json?
```suggestion
                res.Header().Set("content-type", "application/json")
		res.WriteHeader(s.Status.Code)
```